packages/augur-sdk/src/state/create-api.ts

```diff
@@ -147,7 +157,7 @@ export async function createClient(
 
 export async function createServer(config: SDKConfiguration, client?: Augur, account?: string): Promise<{ api: API, controller: Controller, sync: () => Promise<void> }> {
   console.log('Creating Server');
-  // Validate the config -- check that the syncing key exits and use defaults if not
+  // Validate the config -- check that the syncing key exists and use defaults if not
   config = {
     syncing: {
```

Comment:
Should we combine `server` and `syncing` here and just have `syncing` be a part of the server configuration?

Reply:
Are `sdk`, `syncing`, and `server` all related then?

Reply:
yeah -- except `sdk` is kinda like a client side configuration (speaking TO an sdk server) while `server` is for the server from its config....

we can rename all of this to make it make sense

Reply:
Added a bit about this to https://github.com/augurproject/augur/issues/6385.